docs/crio.8.md

```diff
@@ -127,10 +127,10 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
     configuration file named '00-default' has a lower priority than a file
     named '01-my-overwrite'.
     The global config file, provided via '--config,-c' or per default in
-	/etc/crio/crio.conf, always has a lower priority than the files in the directory specified
-	by '--config-dir,-d'.
-	Beside that, provided command line parameters still have a higher priority
-	than any configuration file. (default: /etc/crio/crio.conf.d)
+    /etc/crio/crio.conf, always has a lower priority than the files in the directory specified
+    by '--config-dir,-d'.
+    Beside that, provided command line parameters still have a higher priority
```

Comment:
nit Beside -> Besides

Reply:
and you could drop 'still' in "still have"

Reply:
done